[Refactor] 保存済フロア読込関連の可読性向上

dis- opened the pull request:
関数への括り方と名称の変更により可読性を向上させる。
動作変更は行わない。

dis- committed:
[Refactor] check_visited_floor()の解体

check* の名称だが実態は保存済フロアの読込及び読込時の処理である。
適当な名称を付けて分割する。

dis- committed:
[Refactor] new_floor_allocation() のリネーム

保存済フロア読込成功時のフロア設置処理であることが明確になるよう allocate_loaded_floor() にリネームする。

habu1010 commented on src/floor/floor-changer.cpp:
```diff
@@ -399,9 +412,9 @@ static void update_floor(PlayerType *player_ptr)
-    bool loaded = false;
     sf_ptr = get_sf_ptr(new_floor_id);
-    check_visited_floor(player_ptr, sf_ptr, &loaded);
+    bool loaded = load_visited_floor(player_ptr, sf_ptr);
```
単純な処理なので、関数化するまでもなく、

```c++
const bool loaded = is_visited_floor(sf_ptr) && load_floor(player_ptr, sf_ptr, 0);
```

で十分読みやすいと思えますがどうでしょうか？個人の好みに近いかもしれないのでお任せしますが、変更しない変数は基本const宣言推奨なのでそちらの方は変えておいたほうがよいかなと思います。

dis- replied:
たしかに、ここまで単純な処理になっていれば関数にするまでもないですね。
そのように修正しました。

dis- committed:
[Refacotr] update_new_floor_feature() の再構成

実際にこの周辺で行われている処理は保存済フロアの読込、読込不可の場合新規フロアの設置から事後処理である。
処理が明確になるよう関数の切り出し範囲の変更及びリネームを行う。

habu1010 reviewed (approved):
確認しましたのでApproveします。

dis- commented:
確認ありがとうございました。
